src/module/addandremove.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+let arr = [];
+arr = JSON.parse(localStorage.getItem("arr")) || [];
+const addRemove = () => {
```

Comment:
- [ ] @SabaAhmad786 great work on your add and remove functionality, however as the previous reviewer requested kindly consider changing the add and delete function are implemented so that they don't refresh the page, but just change the DOM element.' you can separate the display function from the add and delete function.'